pkg/autoscaler/config/autoscalerconfig/autoscalerconfig.go

```diff
@@ -50,6 +50,10 @@ type Config struct {
 	// services. This can be set to 0 iff AllowZeroInitialScale is true.
 	InitialScale int32
 
+	// MinScale is the default min scale for any revision created without an
+	// autoscaling.knative.dev/maxScale annotation
```

Comment:
```suggestion
	// autoscaling.knative.dev/minScale annotation
```

Reply:
fixed, sorry for the typos, went through and made sure there aren't any more